.gitignore

```diff
@@ -4,5 +4,5 @@
 deps/deps.jl
 *.mem
 *.h5
-Manifest.toml
+/Manifest.toml
```

Comment:
Why do we want to check in the docs `Manifest.toml`?

Reply:
Currently only Zygote#master uses ChainRules, so we need to check in the manifest until a release is made (Zygote's release schedule is typically a bit slower, so I'd rather not wait). When Zygote does make a release, I'll open a new PR removing the docs manifest.